src/include/ipc/topology.h

```diff
@@ -48,6 +48,7 @@ enum sof_comp_type {
 	SOF_COMP_DCBLOCK,
 	SOF_COMP_SMART_AMP,		/**< smart amplifier component */
 	SOF_COMP_CODEC_ADAPTOR,		/**< codec adaptor */
+	SOF_COMP_COPIER,		/** Copier for IPC4 */
 	/* keep FILEREAD/FILEWRITE as the last ones */
```

Comment:
@plbossart @cujomalainey didn't we decide that we won't use this anymore. But use UUID instead?

Reply:
We did, this should be removed.

Reply:
Do we have a timeline to delete this enum?

Reply:
updated, removed this definition.